Makefile

```diff
@@ -69,6 +70,12 @@ $(BINARY).linux:
 	# Building linux binary.
 	GOOS=linux go build -o $(BINARY).linux -ldflags "-w -s -X github.com/davidnewhall/unifi-poller/pkg/unifi-poller.Version=$(VERSION)" $(PACKAGE)
 
+dockerbin: $(BINARY).dockerbin
+$(BINARY).dockerbin:
```

Comment:
Instead of making `dockerbin` can we use the `$(BINARY).linux` for the same purpose? The only difference I see here is the addition of `CGO_ENABLED=0` - I'm not sure that has any real affect on the build though.

Reply:
Or perhaps we just stick to `make` in the `Dockerfile` (instead of `make dockerbin`) and let it build on whatever architecture is running?

Reply:
yeah i removed the duplication and made CGO_ENABLED=0 as prefix in the build of dockerfile. 
CGO_ENABLED is mandatory in this case because the resulting image is not debian based - it is from scratch. so this is a minimal docker image ( 6mb ) which can easily be used for go stuff